styles/Languages/go.less

```diff
@@ -38,4 +38,10 @@
         .placeholder {
             color: @color-yellow;
         }
+
+        .string.quoted.double{
```

Comment:
Done. Is there some package that formats less automatically? I have no integration at all in Atom for this. Feels like I am driving with three wheels if you know what I mean.